dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -618,11 +628,26 @@ public void saveAction(final WorkflowAction action,
 				}
 			}
 		} catch (Exception e) {
-			Logger.error(WorkflowAPIImpl.class, e.getMessage(), e);
+			Logger.error(WorkflowAPIImpl.class, e.getMessage());
+			Logger.debug(WorkflowAPIImpl.class, e.getMessage(), e);
 			throw new DotDataException(e.getMessage(), e);
 		}
 	}
 
+	private boolean existsScheme(final String schemeId) {
+
+		boolean existsScheme = false;
+
+		try {
+
+			existsScheme = null != this.findScheme(schemeId);
+		} catch (Exception e) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)